### PR TITLE
Implement color accessibility features

### PR DESCRIPTION
**Pull request by acgrobman**
Also removes construction notice, implements heading, and externalizes cutoff values.

### CODE REVIEW

File: src/components/AqiGauge/AqiDial.tsx [hunk 1]
```diff
@@ -21,40 +23,38 @@ const AqiLabel: ({currentAqi}: DialProps) => JSX.Element = ({
...
     label = t('good');
-  } else if (aqi <= moderate) {
-    [textColor, backgroundColor] = ['black', '#FEFF00'];
+  } else if (aqi <= aqiCutoffs.moderate) {
+    correctColor = currentColorScheme.moderate;
     label = t('moderate');
-  } else if (aqi <= sensitiveGroups) {
-    [textColor, backgroundColor] = ['black', '#FF7E02'];
+  } else if (aqi <= aqiCutoffs.sensitiveGroups) {
+    correctColor = currentColorScheme.sensitive;
```
Comment (akampen) [inline src/components/AqiGauge/AqiDial.tsx:42]: nit: sensitiveGroups and sensitive (aqiCutoffs and currentColorScheme should use the same language)

File: src/components/AqiGauge/GaugeSvg.tsx [hunk 2]
```diff
@@ -72,37 +75,33 @@ const GaugeSvg: ({currentAqi}: DialProps) => JSX.Element = ({
...
-      arcColor = '#FF0202';
-    } else if (aqi <= veryUnhealthy) {
-      arcColor = '#8F3F97';
+    let arcColor = currentColorScheme.good;
+    if (aqi <= aqiCutoffs.good) {
+      arcColor = currentColorScheme.good;
+    } else if (aqi <= aqiCutoffs.moderate) {
+      arcColor = currentColorScheme.moderate;
+    } else if (aqi <= aqiCutoffs.sensitiveGroups) {
+      arcColor = currentColorScheme.sensitive;
```
Comment (akampen) [inline src/components/AqiGauge/GaugeSvg.tsx:84]: Nit: same as above

File: src/components/AqiGraph/AqiGraph.tsx [hunk 3]
```diff
@@ -94,15 +91,15 @@ const AqiGraph: ({sensorDocId}: GraphProps) => JSX.Element = ({
...
                 const newElement: GraphElement = {x: hoursAgo, y: element.aqi};
                 // Add element to data
-                if (element.aqi <= good) {
+                if (element.aqi <= aqiCutoffs.good) {
                   allData.good.push(newElement);
-                } else if (element.aqi < moderate) {
+                } else if (element.aqi < aqiCutoffs.moderate) {
                   allData.moderate.push(newElement);
-                } else if (element.aqi < sensitiveGroups) {
+                } else if (element.aqi < aqiCutoffs.sensitiveGroups) {
```
Comment (akampen) [inline src/components/AqiGraph/AqiGraph.tsx:98]: nit: again

File: src/components/Map/markerStyle.tsx [hunk 4]
```diff
@@ -1,40 +1,38 @@
...
-  } else if (aqi < unhealthy) {
-    color = '"#FF0202"'; // Red
-  } else if (aqi < veryUnhealthy) {
-    color = '"#8F3F97"'; // Purple
+  if (aqi < aqiCutoffs.good) {
+    color = colors.good;
+  } else if (aqi < aqiCutoffs.moderate) {
+    color = colors.moderate;
+  } else if (aqi < aqiCutoffs.sensitiveGroups) {
+    color = colors.sensitive;
```
Comment (akampen) [inline src/components/Map/markerStyle.tsx:24]: nit: mismatch

File: src/components/Util/Colors.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  activeOpacity: number;
+  inactiveOpacity: number;
+};
+
+type Color = {
+  backgroundColor: string;
+  textColor: string;
+};
+
+const EpaColorScheme: ColorScheme = {
```
Comment (akampen) [inline src/components/Util/Colors.tsx:22]: comment mentioning that these colors are "official" and from the EPA

File: src/components/Util/Colors.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  moderate: {backgroundColor: '#FEFF00', textColor: 'black'},
+  sensitive: {backgroundColor: '#FF7E02', textColor: ' black'},
+  unhealthy: {backgroundColor: '#FF0202', textColor: 'white'},
+  veryUnhealthy: {backgroundColor: '#8F3F97', textColor: 'white'},
+  hazardous: {backgroundColor: '#7E0224', textColor: 'white'},
+  activeOpacity: 1,
+  inactiveOpacity: 0.3,
+};
+
+const AccessibleColorScheme: ColorScheme = {
```
Comment (akampen) [inline src/components/Util/Colors.tsx:33]: comment mentioning that we chose these as colorblind-friendly

File: src/components/Util/Colors.tsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  moderate: {backgroundColor: '#91BFDB', textColor: 'black'},
+  sensitive: {backgroundColor: '#E0F3F8', textColor: 'black'},
+  unhealthy: {backgroundColor: '#FEE090', textColor: 'black'},
+  veryUnhealthy: {backgroundColor: '#FC8D59', textColor: 'black'},
+  hazardous: {backgroundColor: '#D73027', textColor: 'white'},
+  activeOpacity: 1,
+  inactiveOpacity: 0.6,
+};
+
+const ColorToggle: React.FC = () => {
```
Comment (akampen) [inline src/components/Util/Colors.tsx:44]: docstring

File: src/contexts/ColorContext.tsx [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+        toggleColorScheme: toggleColorScheme,
+      }}
+    >
+      {children}
+    </ColorContext.Provider>
+  );
+};
+
+/**
+ * Custom hook to allow other components to use authentication status
```
Comment (akampen) [inline src/contexts/ColorContext.tsx:58]: authentication status?

File: src/util.tsx [hunk 9]
```diff
@@ -9,4 +9,18 @@ function validData(value: string | boolean, type: string): boolean {
...
+/**
+ * Upper bounds on AQI categories.
+ *
+ * Source: https://www.airnow.gov/aqi/aqi-basics/
+ * Anything above 300 is considered "Hazardous"
+ */
+const aqiCutoffs = {
+  good: 50, // Air quality is good (0-50)
+  moderate: 100, // Air quality is acceptable (51-100)
+  sensitiveGroups: 150, // Health risk for sensitive groups (101-150)
```
Comment (akampen) [inline src/util.tsx:21]: nit: switch to sensitive

File: src/contexts/ColorContext.tsx (hunk 8 above)
Comment (acgrobman) [reply src/contexts/ColorContext.tsx:58]: Whoops. Left over from copy paste oops

**Review by CarsonHerness — APPROVED** [review]
A few minor comments but this looks great, nice job!

**Pull request merged**